### PR TITLE
Login page displays when un-logged User clicks on 'Book ticket' tab

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/test/java/tests/BookTicketTests.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import page_object.LoginPage;
+
+public class BookTicketTests extends BaseTest {
+
+    BookTicketPage bookTicketPage = new BookTicketPage();
+    LoginPage loginPage = new LoginPage();
+    String headerLogin = "Login Page";
+
+    @BeforeMethod
+    public void clickBookTicketTab() {
```
Comment (tanmle) [inline src/test/java/tests/BookTicketTests.java:17]: you should just name it precondition

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -12,11 +13,15 @@
     String errorLoginFormMsg = "There was a problem with your login and/or errors exist in your form.";
     String errorPasswordMsg = "Invalid username or password. Please try again.";
 
-    @Test(description = "User can log into Railway with valid username and password")
-    public void tc001_LoginWithValidUsernameAndPassword() {
+    @BeforeMethod
+    public void clickLoginTab(){
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:17]: you should just name it precondition

Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: changed method name

File: src/test/java/tests/BookTicketTests.java (hunk 1 above)
Comment (Baonganbui123) [reply src/test/java/tests/BookTicketTests.java]: changed method name

**Review by tanmle — APPROVED** [review]

File: src/test/java/tests/BookTicketTests.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+public class BookTicketTests extends BaseTest {
+
+    BookTicketPage bookTicketPage = new BookTicketPage();
+    LoginPage loginPage = new LoginPage();
+    String headerLogin = "Login Page";
+
+    @BeforeMethod
+    public void precondition() {
+
```
Comment (MinhQuan119) [inline src/test/java/tests/BookTicketTests.java:18]: Remove redundant rows

Comment (Baonganbui123) [reply src/test/java/tests/BookTicketTests.java]: Removed

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**